grpc/interceptors.go

```diff
@@ -18,12 +18,14 @@ import (
 
 	"github.com/letsencrypt/boulder/cmd"
 	berrors "github.com/letsencrypt/boulder/errors"
+	"github.com/letsencrypt/boulder/web"
```

Comment:
It feels a little odd to have a dependency going from grpc to web. I think it's probably the right thing to do, given the existence of //web/context.go, but it does make me think that file should maybe be moved to its own package someday.